### PR TITLE
Tic tac toe

### PR DESCRIPTION
**Pull request by jcromerohdz**
This pull request has:
- Game Board functionality
- Game Flow functionality
- Dom functionality
- Player functionality
- Tic Tac Toe UI HTML index .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          <div class="boxcontainer">
+            <div class="example" data-id="0">O</div>
+            <div class="example" data-id="1">O</div>
+            <div class="example" data-id="2">O</div>
+            <div class="example" data-id="3">X</div>
+            <div class="example" data-id="4">X</div>
+            <div class="example" data-id="5">X</div>
+            <div class="example" data-id="6">O</div>
+            <div class="example" data-id="7">O</div>
+            <div class="example" data-id="8">O</div>
```
Comment (Oghenebrume50) [inline index.html]: These divs should be created without content

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED

Hi Team good job on this one, everything works fine and it looks professional, please fix the one issue outline below and **request for another review**  :slightly_smiling_face: 

**RAPHAEL NORIODE** :sunglasses:

File: index.html (hunk 1 above)
Comment (jcromerohdz) [reply index.html]: Thanks Raphael @Oghenebrume50  for taking the time to review our project we change the comment above as you requested.

**Review by bellom — APPROVED** [review]
## Hi Team :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**